src/content.js

```diff
@@ -272,6 +410,7 @@ Function used to avoid multiple injection (cleaner than using an if?)
                 textbox.remove();
             })
             ptr.img.removeEventListener('load', onImageReload);
+            processedImages.delete(ptr.img);
```

Comment:
```suggestion
            processedImages.delete(ptr.img);
            processingImages.delete(ptr.img);
```

Think here we should remove it from both sets?